3-Isomorphism.tex

```diff
@@ -235,7 +235,7 @@ \section{Reasoning with Arrows}
 When two objects $a$ and $b$ are isomorphic, any isomorphism $f$ induces a one-to-one mapping $(f \circ -)$ between corresponding sets of arrows.  
 
 
-Suppose that we don't know if the objects are isomorphic, but we know that there is an invertible mapping, $\alpha_x$, between sets of arrows impinging on $a$ and $b$ from every object $x$. In other words, for every $x$, $\alpha_x$ is a bijection of arrows. The answer is ``yes'', as long as the family $\alpha_x$ satisfy the naturality condition.
```

Comment:
Moved this sentence below, as it didn't make sense to have answer *before* the question being asked.
Also changed "satisfy" -> "satisfies" (because it's singular - family satisfies)